### PR TITLE
update BTSbot to v2, which reads channels-first and emits logits

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR updates BTSbot to v2, which reads channels-first and emits logits.

### CODE REVIEW

**Review by thomasculino — APPROVED** [review]
LGTM
